executor/importer/import.go

```diff
@@ -654,6 +666,25 @@ func (p *Plan) initOptions(seCtx sessionctx.Context, options []*plannercore.Load
 	if _, ok := specifiedOptions[disableTiKVImportModeOption]; ok {
 		p.DisableTiKVImportMode = true
 	}
+	if opt, ok := specifiedOptions[cloudStorageURIOption]; ok {
+		v, err := optAsString(opt)
+		if err != nil {
+			return exeerrors.ErrInvalidOptionVal.FastGenByArgs(opt.Name)
+		}
+		// set cloud storage uri to empty string to force uses local sort when
+		// the global variable is set.
+		if v != "" {
+			b, err := storage.ParseBackend(v, nil)
+			if err != nil {
+				return exeerrors.ErrInvalidOptionVal.FastGenByArgs(opt.Name)
+			}
+			// only support s3 and gcs now.
+			if b.GetS3() == nil && b.GetGcs() == nil {
```

Comment:
@wjhuang2016 we will return error when it's google cloud storage? I remember we have some S3 only implementation. Which release version do we plan to remove the restriction?

Reply:
in doc, import-into only support s3 too, see https://github.com/pingcap/docs-cn/pull/15021/files#diff-218ca9ffe9f1f361d40ff3e8ff7ea091223220e4193574191cf763a64a5de299R129

but for integration test, we need gcs